Userland/Libraries/LibJS/Runtime/Object.cpp

```diff
@@ -190,6 +190,18 @@ bool Object::create_data_property_or_throw(PropertyName const& property_name, Va
     return success;
 }
 
+// 7.3.6 CreateNonEnumerableDataPropertyOrThrow ( O, P, V ), https://tc39.es/proposal-error-cause/#sec-createnonenumerabledatapropertyorthrow
```

Comment:
flaw in the spec: 7.3.6 already is something else :^)
(no worries, keep it as is)